content/docs/0.16.x/integrations/webhooks/jenkins/index.md

```diff
@@ -0,0 +1,142 @@
+you can even provide information back to Keptn (e.g., `test.finished` with `result=fail`).
+
+*Note*: The instructions on this page assume that you do **not** have `jmeter-service` installed.
+
+## Configure Jenkins
+
+*Note*: Keptn needs to reach your Jenkins installation (either directly or via a proxy). This goes way beyond our documentation, therefore we suggest reaching out to your Jenkins administrator.
+
+* Open your Jenkins UI, log in as an administrator, and configure the pipeline you want to call
+* Select the pipeline you want to call, and click on "Configure".
+* Select "Build triggers" and tick "Trigger builds remotely"
+* Create an authentication token (e.g., using a local password manager) - you will need this token later when creating the webhook in Keptn
+* Also note down the URL shown below (e.g., `http://jenkins.127.0.0.1.nip.io/jobs/build?token=`) - you will need it later when creating the webhook in Keptn
+
+  {{< popup_image
+  link="./assets/jenkins-trigger-builds-remotely.png"
+  caption="Enable Trigger builds remotely on Jenkins"
+  width="700px">}}
+
+## Store sensitive data in a secret
+
+To secure the sensitive data of your Jenkins webhook URL, a secret needs to be created:
+
+* Go to **Uniform page** > **Secret** and click the **Add Secret**
+* Enter a *name* (e.g., `jenkins-secret`) and select `keptn-webhook-service` as a secret *scope*
+* Enter a unique name for the *key* (e.g., `my-pipeline-secret`)
+* Copy-paste the authentication token from the previous step into the *value* field:
+
+  {{< popup_image
+  link="./assets/jenkins-uniform-secret.png"
+  caption="Create a secret for storing the Jenkins authentication token"
+  width="700px">}}
+
+
+
+## Set up Jenkins integration via Webhook
+
+To create a webhook integration, a subscription needs to be created:
+
+* Go to **Uniform page** > **Uniform**, select the *webhook-service*, and click the **Add subscription** button.
+
+* For this integration, we would like trigger a Jenkins pipeline when a *test* task in the *dev* and *staging* stage is triggered. Therefore, you need to select:
+    * *Task*: `test`
+    * *Task suffix*: `triggered`
+    * *Filter*: `Stage:dev`, `Stage:staging`
+
+
+* Once the above-configured event gets fired, the Jenkins pipeline has to be triggered. Therefore, you need to select/enter:
+    * *Request method*: `GET`
+    * *URL*: The webhook URL from above: `http://jenkins.127.0.0.1.nip.io/jobs/build?token=`
+    * Reference the secret to add the webhook identifier at the end of the URL. Therefore, clicking on the *key* icon, select the secret `jenkins-secret` and the key `my-pipeline-secret`. This will reference the secret value containing the sensitive data of your webhook URL: `http://jenkins.127.0.0.1.nip.io/jobs/build?token={{.secret.jenkins-secret.my-pipeline-secret}}`
```

Comment:
```suggestion
    * Reference the secret to add the webhook identifier at the end of the URL by clicking on the *key* icon, select the secret `jenkins-secret` and the key `my-pipeline-secret`. This will reference the secret value containing the sensitive data of your webhook URL: `http://jenkins.127.0.0.1.nip.io/jobs/build?token={{.secret.jenkins-secret.my-pipeline-secret}}`
```